### ISSUE TITLE
Add support for opt-in panic hooking

### ISSUE THREAD

**Issue by Jake-Shadle** [EmbarkStudios/crash-handling#78]
It'd be nice to allow users to opt-in to panic handling that could also be reported with a minidump the same as crashes.

**Comment by ReactorScram** [comment]
Yeah right now I'm seeing that if I panic at any point after setting up crash handling, it leaks the server process? Maybe I've set something up wrong?

**Comment by ReactorScram** [comment]
nvm just found the docs for stale connections https://docs.rs/minidumper/latest/minidumper/struct.Server.html#method.run
